tests/Feature/ComponentSecurityTest.php

```
@@ -14,13 +14,13 @@ class ComponentSecurityTest extends TestCase
     public function it_prevents_command_injection_in_component_discovery()
     {
         $discovery = app(StandaloneComponentDiscovery::class);
-        
+
         // Create a malicious component directory
         $maliciousDir = base_path('components/core/evil; rm -rf /');
```

PHPMD (2.15.0)

Avoid unused local variables such as '$maliciousDir'. (Unused Code Rules)

(UnusedLocalVariable)